src/components/LoadingScreen.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import loadingSpinner from '../assets/dumbbell_loading_spinner.svg';
```

Comment:
Try to import from `assets/index.tsx`, not directly.
Example - https://github.com/Min-Ho-Lim/Gym-Landing-Page/blob/main/src/components/BlogPosts.tsx#L2